oeps/oep-0001.rst

```diff
@@ -232,6 +246,15 @@ OEPs can also be superseded by a different OEP, rendering the original
 obsolete. In that case, the OEP's status should be changed to "Replaced"
 and updated with a link to its superseding OEP.
 
+Obsolete
```

Comment:
I'm not really opposed to this, but I wonder if it is a problem simply because we don't have good index-like docs to what is relevant, so someone might still feel compelled to review all OEPs.

Reply:
A follow-up to this that I would like to do is to organize the OEPs  a bit better in the index by what kind of OEPs they are but as a part of that I wanted to move the obsolete ones out of the way.